modules/site-icon.php

```diff
@@ -1,17 +0,0 @@
-<?php
```

Comment:
I'm not sure the rationale (prevent fatals from an `include` cached?), but previously deprecated modules still have their `{$slug}.php` file present with a comment saying deprecated.